schemas/layer-4.cue

```diff
@@ -1,35 +1,45 @@
 // Top level schema //
-
-"evaluation-plans": [...#EvaluationPlan]
+#FrameworkEvaluation: {
+    // name of the framework being evaluated
+    name: string
+    // ID of the framework being evaluated
+    frameworkID: string
+    // one or more evaluations of the framework controls
+    evaluations: [#ControlEvaluation, ...]
+}
 
 // Types
 
+#URL: =~"^https?://[^\\s]+$"
+
 #ControlEvaluation: {
+    // name of the control being evaluated
     name: string
-    "control-id": string
-    result: #Result
-    message: string
```

Comment:
In the absence of documentation or existing usage, `message` should go away.

Reviewers - Please provide any such examples if you are aware of them